src/netex_france/offer.rs

```diff
@@ -615,7 +615,7 @@ impl<'a> OfferExporter<'a> {
             return Ok(None);
         }
         let coord_epsg2154 = self.converter.convert(*coord)?;
-        let coord_text = Node::Text(format!("{} {}", coord_epsg2154.x(), coord_epsg2154.y()));
+        let coord_text = Node::Text(format!("{} {}", coord_epsg2154.lon, coord_epsg2154.lat));
```

Comment:
The `proj` API is now trait based. 

Before:
pass in your native type, which gets converted to a geo_types::Coord
returns a geo_types::Coord

now:
pass in your native type
return your native type

Achieved by conforming your native type to the new proj::Coord trait.